tests/anmykh/test_syntax_error.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def test_syntax_error():
+    sad = {"abc": 123}
+    with pytest.raises(SyntaxError):
+        sad{"abc"}
+
+def test_syntax_error2():
+   False = 0
+   with pytest.raises(SyntaxError):
+       False
```

Comment:
error happens in line 9, but not in line 11

Reply:
fixed